noir-projects/noir-contracts/contracts/token_contract/src/main.nr

```diff
@@ -323,8 +322,15 @@ contract Token {
         }
 
         let from_keys = get_public_keys(from);
+        // TODO: constrain encryption below - we are using unconstrained here only becuase of the following Noir issue
+        // https://github.com/noir-lang/noir/issues/5771
```

Comment:
I think this is an okay way to unblock this for now. Also the code will get refactored a lot with partial notes so it might even go away